### ISSUE TITLE
Add storage object create form in dev console

### ISSUE THREAD

**Issue by oscargoldman** [heroiclabs/nakama#342]
It would be pretty handy to be able to create objects directly in the dev console without having to import.

**Comment by novabyte** [comment]
@oscargoldman Nice suggestion 👍
In the meantime we do have a way to import storage objects in JSON or CSV format into the game server. These objects will become owned by the system user.

**Comment by oscargoldman** [comment]
yup, have been using it to import our existing schema. Took me a while to get the scripts to escape the value node properly but it's working now.

**Comment by jpntex** [comment]
Added in [v3.30.0](https://github.com/heroiclabs/nakama/releases/tag/v3.30.0)
